doc/source/whatsnew/v0.25.0.rst

```diff
@@ -617,13 +617,12 @@ Strings
 - Improved error message when passing :class:`Series` of wrong dtype to :meth:`Series.str.cat` (:issue:`22722`)
 -
 
-
 Interval
 ^^^^^^^^
 
 - Construction of :class:`Interval` is restricted to numeric, :class:`Timestamp` and :class:`Timedelta` endpoints (:issue:`23013`)
 - Fixed bug in :class:`Series`/:class:`DataFrame` not displaying ``NaN`` in :class:`IntervalIndex` with missing values (:issue:`25984`)
--
+- :class:Interval will complain when creating point interval with an inconsistent `closed` value (:issue:`26893`)
```

Comment:
this is not a very friendly comment, can you re-word, put double backticks around ``closed=`. This is also not clear what you are actually fixing here. What is an inconsistent 'closed' value?

Reply:
I'm talking about `(0,0]` or `[0,0)` when the same point is both open and closed, which makes no sense. Can you suggest a better wording?

Reply:
Reworded with explicit cases